bleu/op-bacchus
# Jean/op 173 connect to eas subgraph to list attestations

Add list attestations page with no developed UI.

How to test:
- access /events
- login in wallet
- switch to optimism sepolia
- create some event attestaitons
- the attestations should automatically appear in the page, just like the page below:

![image](https://github.com/user-attachments/assets/b8fa4cdb-b183-4330-9cf2-0ea41650a128)
